usaspending_api/download/helpers/download_annotations_helper.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.postgres.aggregates import StringAgg
+from django.db.models.functions import Concat
+from django.db.models import Value
+
+
+def universal_transaction_matview_annotations():
+    federal_account_query_path = "transaction__award__financial_set__treasury_account__federal_account"
+    annotation_fields = {
+        "federal_accounts_funding_this_award": StringAgg(
+            Concat(
+                "{}__agency_identifier".format(federal_account_query_path),
+                Value("-"),
+                "{}__main_account_code".format(federal_account_query_path),
+            ),
+            ";",
+            distinct=True,
+        )
+    }
+    return annotation_fields
+
+
+def universal_award_matview_annotations():
+    federal_account_query_path = "award__financial_set__treasury_account__federal_account"
+    annotation_fields = {
+        "federal_accounts_funding_this_award": StringAgg(
+            Concat(
+                "{}__agency_identifier".format(federal_account_query_path),
+                Value("-"),
+                "{}__main_account_code".format(federal_account_query_path),
+            ),
+            ";",
+            distinct=True,
+        )
+    }
+    return annotation_fields
```

Comment:
can we rename this to "download_annotations_library" or have these stored in the dict or something? I've never been a big fan of "helper" classes, and this isn't really a utils library, it's part of the business logic.

Reply:
I might actually advise putting it into the historical lookups file, since those two are so tightly coupled.

Reply:
I personally like the idea of keeping them separate but agree I should change the name. I think it will help as either of those two files continue to grow. They are tightly coupled, but not directly only through csv_source and csv_generation.